xml/api/resources-2.25.rng

```diff
@@ -107,6 +116,9 @@
             <optional>
                 <element name="xml"> <text/> </element>
             </optional>
+            <optional>
+                <attribute name="description"> <data type="string" /> </attribute>
+            </optional>
```

Comment:
It would be better to move this above the "xml" element with the other attributes